### PR TITLE
[watchkit] Update for beta 1 to 3 (mostly for watchOS)

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by spouliot** [comment]
WatchKit.WKInterfaceInlineMovie : Handle
WatchKit.WKInterfacePaymentButton : Handle

It could be missing frameworks (e.g. avfoundation and passkit). Need to confirm on devices and Xcode.

File: src/WatchKit/WKDefs.cs [hunk 1]
```diff
@@ -165,4 +165,47 @@ public enum WKInterfaceSemanticContentAttribute : nint
...
+		Changed,
+		Ended,
+		Cancelled,
+		Failed,
+		Recognized
+	}
+
+	[Watch (3,0)][NoiOS]
+	[Native]
+	public enum WKSwipeGestureRecognizerDirection : nuint {
```
Comment (dalexsoto) [inline src/WatchKit/WKDefs.cs]: Maybe missing a `[Flags]` ?

File: src/watchkit.cs [hunk 2]
```diff
@@ -913,4 +989,250 @@ interface WKAudioRecorderControllerOptionsKey {
...
+
+#if false // FIXME SceneKit not yet enabled on platform
+	[Watch (3,0)][NoiOS]
+	[BaseType (typeof (WKInterfaceObject))]
+	interface WKInterfaceSCNScene : ISCNSceneRenderer {
+
+		[NullAllowed, Export ("scene", ArgumentSemantic.Retain)]
+		SCNScene Scene { get; set; }
+
+		UIImage GetSnapshot ();
```
Comment (dalexsoto) [inline src/watchkit.cs]: I know this is inside an `false` but maybe adding a Export or a comment here so we do not miss on a future review

Comment (spouliot) [inline src/watchkit.cs]: There's a `FIXME` in the comment flowing `#if false`

File: src/WatchKit/WKDefs.cs (hunk 1 above)
Comment (spouliot) [inline src/WatchKit/WKDefs.cs]: good catch

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by spouliot** [comment]
failures are unrelated

**Pull request merged**